tasks/task01/Narcissistic_Number.py

```diff
@@ -0,0 +1,16 @@
+for k in range(1, 1000) :
+    if(k>=1 and k<10):
+        digit=k%10
+        if digit**1==k:
+         print ('Narcissistic__Number', k)
+    if(k>=10 and k<100):
+            digit1=k//10
```

Comment:
```suggestion
            digit1 = k // 10
```
надо везде добавить пробелов. Есть такой стандарт написания кода в Python - PEP8. Там описано как улучшить код что бы он хорошо читался. Так же советую поставить себе [линтовщик](https://realpython.com/python-code-quality/#linters)